sdk/eventhub/Azure.Messaging.EventHubs/tests/Core/ConnectionTypeExtensionTests.cs

```diff
@@ -42,5 +42,19 @@ public void GetUriSchemeUDisallowsUnknownConnectionTypes()
             var invalidConnectionType = (TransportType)Int32.MinValue;
             Assert.That(() => invalidConnectionType.GetUriScheme(), Throws.ArgumentException);
         }
+
+        /// <summary>
+        ///   Verifies functionality of the <see cref="TransportTypeExtensions.IsWebSocketTransport" />
+        ///   method;
```

Comment:
nit: it should end in a `.` instead of a `;`. There are other occurrences in the same file.